src/web_app/js/port.js

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+
+/* More information about these options at jshint.com/docs/options */
+
+/* globals trace, chrome */
+/* exported Port */
+
+'use strict';
+
+var Port = (function() {
```

Comment:
I thought this is used in background.js, but it appears to be used on the window side. Could you make the name clearer, e.g. WindowPort?